### DISCUSSION TITLE
I found a App Based On UTM on the app store and is available for iOS and iPadOS

### DISCUSSION THREAD

**Discussion by MCES10** [utmapp/UTM#5436] [General]
I was looking in the App Store and I found this app called merge vm professional this seems to based on UTM but has iOS support 
 here is the link (removed) if you look ing the version history you will see what I mean. This is confusing as UTM for iOS was rejected I am confused.
<img width="500" alt="Screenshot 2023-07-07 at 17 59 28" src="https://github.com/utmapp/UTM/assets/138902870/8d914f98-b25d-4ad5-b495-fb2402bf8f94">

**Comment by osy** [comment]
Yes, we’ve contacted apple multiple times about it and they won’t do anything.

**Reply by megakookie** [reply]
dear god, just threaten legal action at this point. Since this code is open source, and that app literally states its from UTM, i would say ask them to give credit. The worst part about this is that they monetized it.

**Reply by MCES10** [reply]
Just put UTM on the app store now that Emulators are allowed

**Reply by bewl87** [reply]
which app is it exactly, i'm intrigued?
